### ISSUE TITLE
Enable Travis CI 

### ISSUE THREAD

**Issue by halton** [01org/node-realsense#5]
@baleboy could you help to setup on Travis CI since you have admin permission.

1. Visit https://travis-ci.org
2. Toggle 01org/node-realsense project.
3. Click the settings (right side of the toggle button in step 3), toggle "Build only if .travis.yml is present".

Then all should works. For more information, please refer to https://docs.travis-ci.com/user/travis-pro/

**Comment by baleboy** [comment]
Apparently I don't have admin rights on the repo, I'll have to request them.

**Comment by baleboy** [comment]
This is done now, please check if it works

**Comment by halton** [comment]
@baleboy thanks a lot!

**Comment by halton** [comment]
![image](https://cloud.githubusercontent.com/assets/433218/23352513/88ec3ec8-fd02-11e6-9ba6-c36a185146e0.png)

@baleboy it shows me it still disabled.

**Comment by baleboy** [comment]
uhm, yes, I was able to configure the "build only if .yml is present" but not the master switch. Let me try again.

**Comment by baleboy** [comment]
Travis and Github were out of sync, now it should *really* be OK

**Comment by halton** [comment]
Now it is working, will close this bug. Thanks a lot for your supporting. @baleboy
